### DISCUSSION TITLE
Array3AreaRemove with index

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#486] [Feature Requests] [answered]
We spoke in a meetup about ops that change arrays to also include an index.
   Array3AreaRemove
   sorting arrays etc.

**Comment by pandrr** [comment, marked as answer]
we solved the issue. 
if we would implement anything we could think of, we would not make any progress ever, because our heads are full of ideas but we dont have a timemachine... 
it would really help if more people started coding. this stuff is not really hard to pull off either

**Reply by marioepsley** [reply]
No problem, I was only mentioning it because you said it was an interesting point.
All good :)
